lib/graphql/schema_comparator/changes.rb

```diff
@@ -99,12 +99,12 @@ def path
       class EnumValueRemoved < AbstractChange
         attr_reader :enum_value, :enum_type, :criticality
 
-        def initialize(enum_type, enum_value)
+        def initialize(enum_type, enum_value, usage)
           @enum_value = enum_value
           @enum_type = enum_type
-          @criticality = Changes::Criticality.breaking(
+          @criticality = usage.input? ? Changes::Criticality.breaking(
             reason: "Removing an enum value will cause existing queries that use this enum value to error."
-          )
+          ) : Changes::Criticality.non_breaking
```

Comment:
Should we customize the reason here to explain why? That it's only used in arguments.

Reply:
Good call since that will help clients understand the distinction between the breaking and non-breaking cases.